starters/nuxt3-pinia-vuetify/nuxt.config.ts

```diff
@@ -0,0 +1,15 @@
+// https://v3.nuxtjs.org/api/configuration/nuxt.config
+export default defineNuxtConfig({
+  modules: [
+    '@pinia/nuxt'
+  ],
+  css: ['vuetify/lib/styles/main.sass'],
+  build: {
+    transpile: ['vuetify']
```

Comment:
Is this better than using `@nuxtjs/vuetify`? Seems like the library is being maintained by the same Nuxt guys